models/models_test.go

```diff
@@ -100,6 +100,18 @@ func (s *ModelsSuite) TestGetUser(c *check.C) {
 	c.Assert(u.Username, check.Equals, "admin")
 }
 
+func (s *ModelsSuite) TestGetUserByUsernameWithExistedUser(c *check.C) {
```

Comment:
Super minor nitpick, can we change these to "WithExisting" instead of "WithExisted"?

Reply:
Thank you for your review! 
I just fixed the names, so please let me know if you have any comments on this PR! 👍